NaviGator/hardware_drivers/navigator_drone_comm/navigator_drone_comm/packets.py

```diff
@@ -0,0 +1,85 @@
+from dataclasses import dataclass
+from enum import Enum
+
+from electrical_protocol import Packet
+
+
+@dataclass
+class HBT(Packet, class_id=0x20, subclass_id=0x00, payload_format=""):
```

Comment:
I would name these `HeartbeatPacket`, etc. [ALL_CAPS is often reserved for constants in Python](https://google.github.io/styleguide/pyguide.html#3164-guidelines-derived-from-guidos-recommendations).